oprf/oprf.go

```diff
@@ -81,7 +81,7 @@ type Client struct {
 type Server struct {
 	suite   *group.Ciphersuite
 	context []byte
-	Kp      *KeyPair
+	Kp      KeyPair
```

Comment:
Why is this not anymore a pointer if the methods will ask for a pointer?

Reply:
Reduces allocation because the pointer can point into the instance of Server.